lms/templates/ux/reference/teams-base.html

```diff
@@ -155,11 +155,218 @@ <h1 class="page-title">Solar Power</h1>
     <div class="page-content">
```

Comment:
@cptvitamin This is the core part of the content, which excludes the page header, breadcrumbs, and some of the area tools like search. Would it make sense to have this be the ```<main>``` element or a higher level that includes those orientation and tools?